File: SEMESTER 2/PHP/armstrong.php
<!DOCTYPE html>
<html>
<head>
    <title>Armstrong Number Checker</title>
</head>
<body>
    <h2>Check if Number is Armstrong</h2>
    <form method="post" action="<?php echo $_SERVER["PHP_SELF"];?>">
        Enter a number: <input type="text" name="number">
        <input type="submit" name="submit" value="Check">
    </form>

    <?php
    if ($_SERVER["REQUEST_METHOD"] == "POST") 
    {
        $number = $_POST['number'];

        $numDigits = strlen($number);
        $sum = 0;

        for ($i = 0; $i < $numDigits; $i++) 
        {
            $sum += pow($number[$i], $numDigits);
        }

        if ($sum == $number) 
        {
            echo "<p>$number is an Armstrong number.</p>";
        } 
        else 
        {
            echo "<p>$number is not an Armstrong number.</p>";
        }
    }
    ?>

</body>
</html>

File: SEMESTER 3/PYTHON/armstrong.py
n = int(input("Enter a number: "))

sum = 0
temp = n

if(n > 999):
    print("Only 3 digit numbers are allowed!")
else:
    while(n > 0):
        r = n % 10
        sum = sum + (r * r * r)
        n = n // 10
    if(temp == sum):
        print("Armstrong number!")
    else:
        print("Not an Armstrong number!")
